src/chrome/content/rules/CadSoft.xml

```diff
@@ -1,16 +1,13 @@
+<!--
+	Non-functional hosts:
+		Mismatch:
+		- cadsoft.de
+-->
 <ruleset name="CadSoft">
 
 	<target host="cadsoft.de" />
 	<target host="www.cadsoft.de" />
-	<!--	* for cross-domain cookie.	-->
-	<target host="*.www.cadsoft.de" />
 
-
-	<securecookie host="^\.?www\.cadsoft\.de$" name=".+" />
-
-
-	<!--	Cert only matches www.
-					-->
 	<rule from="^http://(?:www\.)?cadsoft\.de/"
```

Comment:
Could you update this to use a catching group instead, as recommended in the contributing guide, please?

```suggestion
	<rule from="^http://(www\.)?cadsoft\.de/"
```